src/app/shuffle/page.js

```diff
@@ -68,30 +89,47 @@ const Shuffle = () => {
             {currentVideo?.title}
           </Subhead>
           <>
-            <Button handleClick={() => handleNextClick()}>
-              <Subhead variant='4'>Skip</Subhead>
-            </Button>
             <Button handleClick={() => shuffleVideos()}>
-              <Subhead variant='4'>Shuffle</Subhead>
+              <Subhead variant='5'>Re-Shuffle</Subhead>
             </Button>
+            <div
+              style={{
+                display: 'flex',
+                justifyContent: currentIndex !== 0 ? 'space-between' : 'flex-end',
+                marginTop: '16px',
+                gridColumn: '1/4'
+              }}
+            >
+              {currentIndex !== 0 && (
+                <ControlButton
+                  onClick={() => setCurrentIndex(currentIndex - 1)}
+                  style={{ display: 'flex', flexDirection: 'row', justifyContent: 'center', width: '40%' }}
+                >
+                  <div>
+                    <Subhead variant='3'>Back</Subhead>
+                    <Subhead variant='5'>{shuffledList[currentIndex - 1]?.title}</Subhead>
+                  </div>
+                </ControlButton>
+              )}
+              <ControlButton
+                onClick={() => handleNextClick()}
+                style={{ 
+                  display: 'flex', 
+                  flexDirection: 'row', 
+                  justifyContent: 'center', 
+                  width: '40%',
+                  transition: fadeNextCard ? 'background 5s linear' : 'none',
+                  background: fadeNextCard ? 'white' : 'transparent',
+                }}
```

Comment:
We can move this to the styled component. For example:
`width: ${({ $fullWidth }) => ($fullWidth ? '100%' : 'min-content')};` 

Note the usage of `$` for the prop to denote it as a transient prop and not pass it on to the DOM

https://styled-components.com/docs/basics#passed-props